sdk/src/store.rs

```diff
@@ -3728,16 +3742,22 @@ impl Store {
     ///
     /// asset_type -  mime type of the stream
     /// stream - a readable stream of an asset
-    pub fn load_jumbf_from_stream(asset_type: &str, stream: &mut dyn CAIRead) -> Result<Vec<u8>> {
```

Comment:
This is a published API so it will be a breaking change. I'm ok with that since it is probably little used.